src/backend/web/local/bootstrap.py

```diff
@@ -132,6 +133,22 @@ def update_event(cls, key: EventKey, auth_token: str) -> None:
         event_awards = cls.fetch_event_detail(key, "awards", auth_token)
         list(map(lambda t: cls.store_award(t, event), event_awards))
 
+        # Get insights from prod API
+        # event_oprs = cls.fetch_event_detail(key, "oprs", auth_token)
+        # cls.store_eventdetail(event, "matchstats", event_oprs)
+
+        # Calculate insights locally
+        matchstats_dict = MatchstatsHelper.calculate_matchstats(event.matches)
+
+        # if any([v != {} for v in matchstats_dict.values()]):
+        #     pass
+        # else:
+        #     matchstats_dict = None
+        #     print("Convert the following to a warning!")
+        #     print(f"Matchstat calculation for {event.key} failed!")
+
+        cls.store_eventdetail(event, "matchstats", matchstats_dict)
+
```

Comment:
Should this be uncommented?

Reply:
Leftover debugging code. We need to calculate locally since prod does not calculate component OPRs (yet).